### PR TITLE
Fix #141

### LINKED ISSUE #141: Should sys.getfilesystemencoding() return 'mbcs' now that this is a supported encoding?
_From @ironpythonbot on December 9, 2014 17:37_

CPython does...  
## Work Item Details

**Original CodePlex Issue:**    [Issue 24220](http://ironpython.codeplex.com/workitem/24220)
**Status:** Active
**Reason Closed:**  Unassigned
**Assigned to:**    Unassigned
**Reported on:**    Aug 5, 2009 at 10:40 PM
**Reported by:**    dfugate
**Updated on:** Nov 3 at 8:58 PM
**Updated by:** jdhardy
**Test:**   (CPy) test_support.py

_Copied from original issue: IronLanguages/main#731_

### PR DESCRIPTION
**Pull request by slide**
Unified the error messages with what is printed by CPython

### CODE REVIEW

File: Src/IronPython/Runtime/PythonAsciiEncoding.cs [hunk 1]
```diff
@@ -193,7 +193,7 @@ class NonStrictEncoderFallbackBuffer : EncoderFallbackBuffer {
         private int _index;
 
         public override bool Fallback(char charUnknownHigh, char charUnknownLow, int index) {
-            throw PythonOps.UnicodeEncodeError("'ascii' codec can't encode character '\\u{0:X}{1:04X}' in position {1}", (int)charUnknownHigh, (int)charUnknownLow, index);
+            throw PythonOps.UnicodeEncodeError("'ascii' codec can't encode character '\\u{0:X}{1:04X}' in position {1}: ordinal not in range(128)", (int)charUnknownHigh, (int)charUnknownLow, index);
         }
 
         public override bool Fallback(char charUnknown, int index) {
```
Comment (slozier) [inline Src/IronPython/Runtime/PythonAsciiEncoding.cs]: What about this one?

Comment (slide) [inline Src/IronPython/Runtime/PythonAsciiEncoding.cs]: Not sure what you mean

Comment (slozier) [inline Src/IronPython/Runtime/PythonAsciiEncoding.cs]: There's another `'ascii' codec can't encode character` without `ordinal not in range(128)` in the second Fallback method.

Comment (slide) [inline Src/IronPython/Runtime/PythonAsciiEncoding.cs]: Ah, in the email I got it didn't show that it was for the second one, it just showed the first one.

**Pull request merged**